state-chain/pallets/cf-vaults/src/migrations.rs

```diff
@@ -1,5 +1,9 @@
 use crate::Pallet;
+use cf_runtime_upgrade_utilities::{migration_template, VersionedMigration};
 
-use cf_runtime_upgrade_utilities::{migration_template::Migration, VersionedMigration};
+mod v3;
 
-pub type PalletMigration<T> = VersionedMigration<Pallet<T>, Migration<T>, 2, 3>;
+pub type PalletMigration<T, I> = (
+	VersionedMigration<Pallet<T, I>, v3::Migration<T, I>, 3, 4>,
+	VersionedMigration<Pallet<T, I>, migration_template::Migration<T>, 4, 5>,
```

Comment:
?

Reply:
The idea of the migration template is as a convenience so we don't have to delete/recreate the whole file every time delete a migration. Make the diff easier to read and reduces friction for writing new migrations / deleting old ones.

Reply:
I don't mind removing it but I feel like it's a nice convenience to have - and there's no risk of accidentally 'using' it since it panics.